frontend/src/components/panels/LayerTree.vue

```diff
@@ -72,42 +82,96 @@
 			display: flex;
 			height: 36px;
 			align-items: center;
-			margin: 0 8px;
 			flex: 0 0 auto;
+			position: relative;
+
+			& + .layer-row {
+				margin-top: 2px;
+			}
+
+			.layer-visibility {
+				flex: 0 0 auto;
+				margin-left: 4px;
+			}
+
+			.node-connector {
+				flex: 0 0 auto;
+				width: 12px;
+				height: 12px;
+				margin: 0 2px;
+				border-radius: 50%;
+				background: var(--color-data-raster);
+				outline: none;
+				border: none;
+				position: relative;
+
+				&::after {
+					content: "";
+					position: absolute;
+					width: 0;
+					height: 0;
+					top: 2px;
+					left: 3px;
+					border-style: solid;
+					border-width: 0 3px 6px 3px;
```

Comment:
I would have expected this to be pointed right and the expanded one to be pointed down (like in in vscode, windows file explorer etc)

Reply:
me to deferring to @Keavon